install/installation-redhat.md

```diff
@@ -32,30 +36,31 @@ instead.
 <procedure>
 
 ### Installing self-hosted TimescaleDB on Red Hat-based systems
+
 1.  At the command prompt, as root, add the PostgreSQL third party repository
     to get the latest PostgreSQL packages:
     <terminal>
 
     <tab label='Red Hat'>
 
     ```bash
-    yum install https://download.postgresql.org/pub/repos/yum/reporpms/EL-$(rpm -E %{rhel})-x86_64/pgdg-redhat-repo-latest.noarch.rpm
+    sudo yum install https://download.postgresql.org/pub/repos/yum/reporpms/EL-$(rpm -E %{rhel})-x86_64/pgdg-redhat-repo-latest.noarch.rpm
```

Comment:
Don't put the `sudo` in. You already say "as root" in the stem sentence, so it's fine.

Reply:
I was trying to keep it consistent with the Arch Linux updates that we made recently.

Reply:
We should have it in our style guide that we don't use `$`, `#`, or `sudo`.

Reply:
Thank you @Loquacity !!!

Reply:
done!!! incorporated the changes @Loquacity !!!!